<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>468</width>
    <height>904</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="main_frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>470</width>
     <height>900</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(190, 249, 255);
border-radius: 20</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="quitButton">
    <property name="geometry">
     <rect>
      <x>434</x>
      <y>10</y>
      <width>26</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift Light</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color:  rgb(136, 239, 255);
  border-radius: 13px;

}
QPushButton:hover {
 background-color: rgba(0, 221, 255, 100);
border: 1px solid rgba(0,0,0,100);
}</string>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_userButton">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>10</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift Light</family>
      <pointsize>20</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <stylestrategy>PreferAntialias</stylestrategy>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
}
QPushButton:hover {
font-weight: 700;
}</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_1">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>320</y>
      <width>140</width>
      <height>140</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift Light</family>
      <pointsize>40</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color:  rgb(136, 239, 255);
  border-radius:0;
  border-top-left-radius:20px;

}
QPushButton:hover {
 background-color: rgba(0, 221, 255, 100);

}</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">entry_buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>305</x>
      <y>320</y>
      <width>140</width>
      <height>140</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift Light</family>
      <pointsize>40</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color:  rgb(136, 239, 255);
  border-radius:0;
  border-top-right-radius:20px;

}
QPushButton:hover {
 background-color: rgba(0, 221, 255, 100);

}</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">entry_buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>165</x>
      <y>320</y>
      <width>140</width>
      <height>140</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift Light</family>
      <pointsize>40</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color:  rgb(136, 239, 255);
  border-radius:0;
}
QPushButton:hover {
 background-color: rgba(0, 221, 255, 100);

}</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">entry_buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>460</y>
      <width>140</width>
      <height>140</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift Light</family>
      <pointsize>40</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color:  rgb(136, 239, 255);
  border-radius:0;
}
QPushButton:hover {
 background-color: rgba(0, 221, 255, 100);

}</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">entry_buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>305</x>
      <y>460</y>
      <width>140</width>
      <height>140</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift Light</family>
      <pointsize>40</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color:  rgb(136, 239, 255);
  border-radius:0;
}
QPushButton:hover {
 background-color: rgba(0, 221, 255, 100);

}</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">entry_buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>165</x>
      <y>460</y>
      <width>140</width>
      <height>140</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift Light</family>
      <pointsize>40</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color:  rgb(136, 239, 255);
  border-radius:0;
}
QPushButton:hover {
 background-color: rgba(0, 221, 255, 100);

}</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">entry_buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>600</y>
      <width>140</width>
      <height>140</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift Light</family>
      <pointsize>40</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color:  rgb(136, 239, 255);
  border-radius:0;
}
QPushButton:hover {
 background-color: rgba(0, 221, 255, 100);

}</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">entry_buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>165</x>
      <y>600</y>
      <width>140</width>
      <height>140</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift Light</family>
      <pointsize>40</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color:  rgb(136, 239, 255);
  border-radius:0;
}
QPushButton:hover {
 background-color: rgba(0, 221, 255, 100);

}</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">entry_buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>305</x>
      <y>600</y>
      <width>140</width>
      <height>140</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift Light</family>
      <pointsize>40</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color:  rgb(136, 239, 255);
  border-radius:0;
}
QPushButton:hover {
 background-color: rgba(0, 221, 255, 100);

}</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">entry_buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="pushButton_enter">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>740</y>
      <width>140</width>
      <height>140</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift Light</family>
      <pointsize>40</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color:  rgb(136, 239, 255);
  border-radius:0;
  border-bottom-left-radius:20px;

}
QPushButton:hover {
 background-color: rgba(0, 221, 255, 100);

}
</string>
    </property>
    <property name="text">
     <string>E</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">entry_buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="pushButton_0">
    <property name="geometry">
     <rect>
      <x>165</x>
      <y>740</y>
      <width>140</width>
      <height>140</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift Light</family>
      <pointsize>40</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color:  rgb(136, 239, 255);
  border-radius:0;
}
QPushButton:hover {
 background-color: rgba(0, 221, 255, 100);

}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">entry_buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="pushButton_del">
    <property name="geometry">
     <rect>
      <x>305</x>
      <y>740</y>
      <width>140</width>
      <height>140</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift Light</family>
      <pointsize>40</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color:  rgb(136, 239, 255);
  border-radius:0;
  border-bottom-right-radius:20px;

}
QPushButton:hover {
 background-color: rgba(0, 221, 255, 100);

}</string>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">entry_buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>260</y>
      <width>471</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift Light</family>
      <pointsize>14</pointsize>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="text">
     <string>Enter your PIN-code</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="user_comboBox">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>100</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift Light</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox:drop-down {
                border: 0px;
            }
            QComboBox:down-arrow {
                width: 0px;
                height: 0px;
                border: 0px;
            }
			QComboBox:hover {
				border-radius: 0;
			}</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>90</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift Light</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 255, 127);
border-radius:32px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="passw_lineEdit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>220</y>
      <width>471</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift Light</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <zorder>quitButton</zorder>
   <zorder>pushButton_1</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>pushButton_3</zorder>
   <zorder>pushButton_4</zorder>
   <zorder>pushButton_5</zorder>
   <zorder>pushButton_6</zorder>
   <zorder>pushButton_7</zorder>
   <zorder>pushButton_8</zorder>
   <zorder>pushButton_9</zorder>
   <zorder>pushButton_enter</zorder>
   <zorder>pushButton_0</zorder>
   <zorder>pushButton_del</zorder>
   <zorder>label</zorder>
   <zorder>user_comboBox</zorder>
   <zorder>label_2</zorder>
   <zorder>passw_lineEdit</zorder>
   <zorder>add_userButton</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="entry_buttonGroup"/>
 </buttongroups>
</ui>
